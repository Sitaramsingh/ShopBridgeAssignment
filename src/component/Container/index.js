import React from 'react';
import Proptypes from 'prop-types';

export default function Container(props) {
    return (
        <div className={ props.style === 'large' ? ['offset-md-1 col-md-10 col-sm-12', props.className].join(' '): ["offset-md-2 col-md-8 col-sm-12", props.className].join(' ')}>
            <div className={["col-md-12 col-sm-12", props.innerClassName].join(' ')}>
                {props.children}
            </div>
        </div>
    );
};

Container.propsType = {
    className:  Proptypes.string,
    innerClassName:  Proptypes.string,
    style: Proptypes.string,
};
