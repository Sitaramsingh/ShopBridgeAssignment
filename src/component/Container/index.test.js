import React from 'react';
import { shallow } from 'enzyme';
import Container from './index';


describe('Component should be render ', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Container/>);
    expect(component).toMatchSnapshot();
  });
});