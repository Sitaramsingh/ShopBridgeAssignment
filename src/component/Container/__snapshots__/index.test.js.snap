// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be render  should render correctly in "debug" mode 1`] = `
<div
  className="offset-md-2 col-md-8 col-sm-12 "
>
  <div
    className="col-md-12 col-sm-12 "
  />
</div>
`;
